configure.in

```diff
@@ -7307,17 +7307,6 @@ if test "$wxUSE_HTML" = "yes"; then
     USE_HTML=1
     SAMPLES_SUBDIRS="$SAMPLES_SUBDIRS html html/about html/help html/helpview html/printing html/test html/virtual html/widget html/zip htlbox"
 fi
-if test "$wxUSE_WEBKIT" = "yes"; then
```

Comment:
We should remove `--enable-webkit` option, setting `wxUSE_WEBKIT`, too. Or do we want to keep it (but ignore/warn about it?) for compatibility?

Reply:
Didn't realize this was only for the webkit ctrl. I would think removing it along with the control would be the right thing, as somebody (and that would be a very rare case I assume) using this in his ./configure would get an error which is useful as he might also use that removed control